calblueprint/ogsc
# Joy/signup fixes

# Touch ups on signup flows
-  implement saved states for the second page of user sign-In
- added blue bar to signup pages (WIP)

## TODO (if applicable)

- [x]  Fix blue bar issue on pages with forms


## How to Test/Use PR feature

- go to http://localhost:3000/users/signUp, verify blue bar on left
- on the second page http://localhost:3000/users/signUp/signup2, click back and next a few times, double check that state saves on the second page

## PR Checklist

Does your branch (check if true):

## Screenshots
added blue bar
<img width="1439" alt="Screen Shot 2021-04-03 at 3 25 26 AM" src="https://user-images.githubusercontent.com/34170094/113475738-401c4b00-942c-11eb-8dc2-0ed8125c4e3c.png">

CC: @sydbui
